docs/book/src/developer/architecture/controllers/machine.md

```diff
@@ -93,6 +99,11 @@ The `status` object **may** define several fields that do not affect functionali
 
 * `failureReason` - is a string that explains why a fatal error has occurred, if possible.
 * `failureMessage` - is a string that holds the message contained by the error.
+* `addresses` - is a list of `MachineAddress` which represents host names, external IP addresses, internal IP addresses,
+external DNS names, and/or internal DNS names for the provider's machine instance. `MachineAddress` is
+defined as:
+    - `type` (string): one of `Hostname`, `ExternalIP`, `InternalIP`, `ExternalDNS`, `InternalDNS`
+    - `address` (string)
```

Comment:
Looks good. Just saw that the indentation is off in https://cluster-api.sigs.k8s.io/developer/providers/machine-infrastructure.html

Could you please fix the indentation there? Thank you very much!